service/allocation_logic.py

```diff
@@ -26,7 +26,8 @@ def create_report(report_start_date, report_end_date, user_id=None, allocation_s
         else:
             return data
     else:
-        write_csv(data)
+        return data
+        #write_csv(data)
```

Comment:
Toss this line? I think if we intend to keep `write_csv` around we should make it a new kwarg for `create_report()`.